emcc.py

```diff
@@ -2860,8 +2860,8 @@ def do_binaryen(target, options, wasm_target):
     building.eval_ctors(final_js, wasm_target, debug_info=intermediate_debug_info)
 
   # after generating the wasm, do some final operations
-  if shared.Settings.SIDE_MODULE:
-    webassembly.add_dylink_section(wasm_target, shared.Settings.RUNTIME_LINKED_LIBS)
+  if shared.Settings.RELOCATABLE:
```

Comment:
(sorry for the "basic" questions here)... so RELOCATABLE applies to main modules and side modules?

Reply:
Yes that is correct.

I'm actually planning on renaming `RELOCATABLE` to `DYNAMIC` to give it a better name going forward.

Reply:
Thanks, yeah that's a much better name (even if it's just an internal define)